jarbas/chamber_of_deputies/tests/test_reimbursement_view.py

```diff
@@ -116,12 +128,31 @@ def test_content_with_year_filter(self):
         content = loads(resp.content.decode('utf-8'))
         self.assertEqual(3, len(content['results']))
 
-    def test_content_with_congressperson_name_filter(self):
+    def test_content_with_search_vector_one_field_filter(self):
         get_reimbursement(congressperson_name='FULANO SILVA BELTRANO', applicant_id=221)
         get_reimbursement(congressperson_name='SILVA TESTA NOME', applicant_id=23)
         get_reimbursement(congressperson_name='BELTRANO CHAVES', applicant_id=23)
+
+        create_search_vector()
```

Comment:
It's ok to use the auxiliar function you've created, totally OK. But just in case I think such call wouldn't be troublesome:

```python
call_command('searchvector', stdout=StringIO())
```

In case you considering simplofying the code and using `call_command` straightforward, just be consistent and change line 152 too : ) But, again, just if you like this idea better. The auxiliar function has its pros too : )